IoTConnect/IoTConnect/Info.plist

```diff
@@ -16,7 +16,10 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
-	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Why you need this?

Reply:
There was a HTTP URL for api

Reply:
this need to be conveyed to IoTConnect this is not allowed, I am resolving for now